pkg/controller/config/process/excluder.go

```diff
@@ -97,13 +97,13 @@ func (s *Excluder) IsNamespaceExcluded(process Process, obj runtime.Object) (boo
 	}
 
 	if obj.GetObjectKind().GroupVersionKind().Kind == "Namespace" && obj.GetObjectKind().GroupVersionKind().Group == "" {
-		return exactOrPrefixMatch(s.excludedNamespaces[process], meta.GetName()), nil
+		return exactOrPrefixSuffixMatch(s.excludedNamespaces[process], meta.GetName()), nil
```

Comment:
Nit: PrefixSuffixMatch -> WildcardMatch

Reply:
Thanks for the review!
I just renamed and rebased onto the latest `master` branch 🙂